web/client/src/components/form/TokenForm.jsx

```diff
@@ -18,16 +18,29 @@ const sleep = (time) => new Promise((acc) => setTimeout(acc, time));
  * @return {FormikStepper} component using Formik for creating a token
  */
 export default function TokenForm() {
+  const [secondLabel, setLabel] = useState("");
+  const [tokenType, setTokenType] = useState("");
+
   return (
     <FormikStepper
       initialValues={{
         tokenType: "",
         tokenFormat: "",
         tokenScopes: "",
+        tokenAudience: "",
+        tokenCredentials: "",
       }}
       onSubmit={async () => {
         await sleep(3000);
       }}
+      setSecondLabel={(value) => {
```

Comment:
In future extensions, we would likely need to support multiple parameters for some token types. For example (UAT/STS token type would require both "audience" and "quota project"). Consider using a generic label such as "Parameters" instead of a dynamic label (Scope/Audience) to support multiple parameters in the form.